tsconfig.json

```diff
@@ -1,63 +1,15 @@
 {
   "compilerOptions": {
-    /* Basic Options */
-    "target": "es5",                          /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
-    "module": "commonjs",                     /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
-    "lib": ["es5", "es2017.object"],          /* Specify library files to be included in the compilation. */
```

Comment:
I think ES5 is from around 2009, no reason not to target ES6 these days. It _should_ make the built code a little cleaner/smaller, but I haven't done any metrics to compare

Reply:
Especially considering we're using node 14 (and should move to node 16 now) I think you're right, this should be fine.